src/main/java/nl/mpcjanssen/simpletask/AppWidgetService.kt

```diff
@@ -153,9 +153,7 @@ data class AppWidgetRemoteViewsFactory(val intent: Intent) : RemoteViewsService.
         rv.setTextViewText(R.id.tasktext, ss)
 
         val relAge = getRelativeAge(task, TodoApplication.app)
-        val relDue = getRelativeDueDate(task , TodoApplication.app, ContextCompat.getColor(TodoApplication.app,android.R.color.holo_green_light),
-                ContextCompat.getColor(TodoApplication.app,android.R.color.holo_red_light),
-                true)
```

Comment:
No reason we need to pass all this in since it never changes. Just the task and the app.